vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,104 @@
+def call(Map pipelineParams) {
+    pipeline {
+        agent any
+        parameters {
+            string(name: 'scylla_version', defaultValue: '', description: 'Scylla version to test')
+            string(name: 'base_versions', defaultValue: '', description: 'Base versions')
+            string(name: 'provision_type', defaultValue: '', description: 'Provision type')
+            string(name: 'new_scylla_repo', defaultValue: '', description: 'New Scylla repo')
+            string(name: 'matrix_storage', defaultValue: '', description: 'This parameter is used for trigger with Scylla master version only. It point which kind of test to run: vnodes or tablets. Expected values: master-tablets OR master-vnodes')
```

Comment:
`matrix_storage` is not exactly clear name for what this does

maybe `matrix_label` ? `matrix_selector` ? `test_selector` ?

Reply:
@fruch  maybe "storage_selector"?

Reply:
not sure how the word storage fits here. 

what are are we storing exactly ?

it sound like we are select a storage type for a VM or something like that

even the description doesn't use the word storage